R/dataProcessPlots.R

```diff
@@ -491,4 +602,140 @@ dataProcessPlots = function(
     }
     .saveTable(result, address, "ConditionPlot_value")
   }
+  if (isPlotly) {
+      return(plots)
+  }
+}
+
+#' converter for plots from ggplot to plotly
+#' @noRd
+.convert.ggplot.plotly = function(plot, tips = "all") {
+    converted_plot <- ggplotly(plot,tooltip = tips)
+    converted_plot <- converted_plot %>% 
+        plotly::layout(
+            width = 800,   # Set the width of the chart in pixels
+            height = 600,  # Set the height of the chart in pixels
+            title = list(
+                font = list(
+                    size = 18
+                )
+            ),
+            xaxis = list(
+                titlefont = list(
+                    size = 15  # Set the font size for the x-axis label
+                )
+            ),
+            legend = list(
+                x = 0,     # Set the x position of the legend
+                y = -0.25,    # Set the y position of the legend (negative value to move below the plot)
+                orientation = "h",  # Horizontal orientation
+                font = list(
+                    size = 12  # Set the font size for legend item labels
+                ),
+                title = list(
+                    font = list(
+                        size = 12  # Set the font size for the legend title
+                    )
+                )
+            )
+        ) 
+    return(converted_plot)
```

Comment:
return is not necessary

Reply:
Commit [remove return statements](https://github.com/Vitek-Lab/MSstats/pull/107/commits/f9695cb099b2ff0f094132d00932ced607fd91bd)